bcipy/bci_tasks/rsvp/calibration.py

```diff
@@ -103,6 +108,24 @@ def execute(self):
                                       self.wait_screen_message_color):
                     break
 
+                #Take a break every number of trials defined in parameters.json
+                if self.enable_breaks:
+                    if not self.trials_before_break == 0:
+                        #Check whether any trials have taken place, and, if so,
+                        #whether the number of trials performed is divisible
+                        #by the number of trials before a break set in parameters
+                        if (idx_o != 0) and (idx_o % self.trials_before_break) == 0:
+                            #Update countdown every second
+                            for counter in range(0,self.break_len):
+                                time = self.break_len - counter
+                                message = f'{self.break_message} {time}s'
+                                self.rsvp.update_task_state(
+                                    text=(message),
+                                    color_list=task_color[idx_o])
+                                self.rsvp.draw_static()
```

Comment:
We generally don't flip or draw static inside our tasks. 

Let's make a helper function in `bci_task_related` like we do for the `get_user_input`. 

It should be called if breaks are enabled and take in the current index, trials before break, rsvp display, and break length. Then we can just import the function everytime we need it